examples/istioctl-analyze/README.md

```diff
@@ -0,0 +1,37 @@
+# Istioctl Analyze
+
+The `istioctl-analyze` config function detects potential issues with your Istio
+configuration and output errors as results. This example invokes the
+istioctl-analyze function using declarative configuration.
+
+## Function invocation
+
+The function is invoked using the function configuration in
+`functions/fn-config.yaml`.
```

Comment:
The command below is sufficient, this sentence is extraneous. See step 3 above.

Reply:
Removed